docs/_documentations/troubleshooting.md

```diff
@@ -458,6 +458,21 @@ Codewind displays an error. In VS Code, the error appears in the Codewind log:
 2. Remove and add the repository back into Codewind with **Manage Template Sources**.
 3. Rebind the project to Codewind.
 
+<!--
+Codewind version: 0.9.0
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/938
+-->
+## Using Appsody stacks images from private Docker registries
+
+**Local scenario**
+For Codewind to work with an Appsody stack image on a private Docker registry, the stack must fully qualify the image name in the `.appsody-config.yaml` configuration of its template.
```

Comment:
Can we keep the example of the format from before?

Also the part about having to do `docker login` to the private registry is gone, we need that.